spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/JaxRsClientMethodBuilderSpec.groovy

```diff
@@ -1205,4 +1255,56 @@ DATA
 			methodBuilderName                                    | methodBuilder
 			"JaxRsClientJUnitMethodBodyBuilder"                  | { org.springframework.cloud.contract.spec.Contract dsl -> new JaxRsClientJUnitMethodBodyBuilder(dsl, properties) }
 	}
+
+	def "should generate test for cookies with string value in JAX-RS JUnit test"() {
+		given:
+			MethodBodyBuilder builder = new JaxRsClientJUnitMethodBodyBuilder(contractDslWithCookiesValue, properties)
+			BlockBuilder blockBuilder = new BlockBuilder(" ")
+		when:
+			builder.appendTo(blockBuilder)
+			def test = blockBuilder.toString()
+		then:
+			test.contains('''.cookie("cookie-key", "cookie-value")''')
```

Comment:
We have a `SyntaxChecker.tryToCompile(methodBuilderName, blockBuilder.toString())`. Maybe it's worth running to see if your code compiles? Check `org.springframework.cloud.contract.verifier.builder.MethodBodyBuilderSpec#should work with execute and arrays [#methodBuilderName]` for reference. The same applies to other tests.